src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -651,27 +651,26 @@ function disableRow(elem, row) {
     $tdElements.parent().hide();
 }
 
-function feedbackRecipientUpdateVisibilityOptions(elem) {
-    $elem = $(elem);
-    var $recipientType = $elem.val();
-    if (isRecipientsTeamMembersValidVisibilityOptionForRecipientType($recipientType)) {
+function feedbackRecipientUpdateVisibilityOptions(recipientType) {
```

Comment:
? This parameter name should actually not be the same as the one in the function below